api/api/views/media_views.py

```diff
@@ -202,7 +226,9 @@ def get_media_results(
 
     # Extra actions
 
+    # stats can always be cached
     @action(detail=False, serializer_class=ProviderSerializer, pagination_class=None)
+    @method_decorator(cache_control(max_age=ttl(days=3), public=True))
```

Comment:
Would we reset cache in Cloudflare after adding a new provider/running data ingestion?

Reply:
Yes, but 3 days is, I think, a pretty acceptable latency in that respect, considering it already takes a week to get new data anyway :shrug: Cache purges in Cloudflare cost money if you do them often, so it's better to just find a value that works. We can adjust as needed.